### DISCUSSION TITLE
代码演示不支持setup语法糖

### DISCUSSION THREAD

**Discussion by PeerQuan666** [vuepress-theme-hope/vuepress-theme-hope#3394] [Q&A] [answered]
![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/142877881/d3e67980-687a-4a79-9f74-161b80e8de3b)

这样写出不来！

**Comment by Mister-Hope** [comment]
是的，建议使用vue playground

**Reply by PeerQuan666** [reply]
那怎么引用自定义组件进来演示呢 比入ElementUI

**Reply by FuckDoctors** [reply, marked as answer]
使用 `playground`, `service` 指定 `https://element-plus.run/` 可使用 Element UI

**Reply by PeerQuan666** [reply]
自己瞎折腾了一番，参考了element的写法抄了一份代码过来
![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/142877881/a3c0482b-094e-4ab3-9c2d-99a7f83c600f)
![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/142877881/b69170b3-09c2-4bc0-a99f-2c88c2abc7c5)
有人有更好更简便的方法也可以提供下
